From 65fad529229d37bc322c4995e93d65ce0f9174f9 Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Tue, 31 May 2011 10:09:12 -0300
Subject: [PATCH] defaults.make: allow BINDIR and LIBDIR to be overriden

When building a toolchain sometimes is important to define where to
the binaries and libraries ought to be placed.

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 defaults.make |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/defaults.make b/defaults.make
index 95a069d..30bded3 100644
--- a/defaults.make
+++ b/defaults.make
@@ -45,8 +45,8 @@ DESTDIR ?=
 PREFIX ?= /usr/local
 BRANCHNAME ?= $(shell sh identify-branch.sh $(SRCDIR))
 
-BINDIR = $(PREFIX)/bin
-LIBDIR = $(PREFIX)/lib
+BINDIR ?= $(PREFIX)/bin
+LIBDIR ?= $(PREFIX)/lib
 SHAREDIR = $(PREFIX)/share
 DATADIR = $(SHAREDIR)/$(PROGRAM_PREFIX)chicken$(PROGRAM_SUFFIX)
 TOPMANDIR = $(SHAREDIR)/man
-- 
1.7.7

